x/launch/genesis.go

```diff
@@ -28,6 +39,14 @@ func ExportGenesis(ctx sdk.Context, k keeper.Keeper) *types.GenesisState {
 	genesis := types.DefaultGenesis()
 
 	// this line is used by starport scaffolding # genesis/module/export
+
+	// Get all request
+	requestList := k.GetAllRequest(ctx)
+	for _, elem := range requestList {
+		elem := elem
```

Comment:
```suggestion
```

Reply:
https://github.com/tendermint/spn/pull/179#discussion_r674209069

Reply:
Please check https://github.com/tendermint/spn/pull/179#discussion_r677302544.

Reply:
Good observation.
I created a PR to look at this change once all the structures for the milestone 1 are initialized